### PR TITLE
Replacing bigtable-protos-internal jar with generated java.

### PR DESCRIPTION
**Pull request by sduskis**
The bigtable-protos project was causing a problem with the release strategy.  The release plugin doesn't allow -SNAPSHOT dependencies, and the protos jar was a -SNAPSHOT version.  Instead of just checking in a non SNAPSHOT version, I think that it's better to check in java files generated from gradle's generateProto process.  That simplified the pom.xml and seems more transparent.

### CODE REVIEW

**Comment by sduskis** [comment]
@kevinsi4508, can you please take a look?  I will pass this on to Angus later today.  Mostly, it would be great to get your opinion on the documentation.

File: bigtable-protos/README.md [hunk 1]
```diff
@@ -54,4 +54,12 @@ Retrieved from https://github.com/google/googleapis:
...
 
+# Generating the java gRPC implementations
 
+The gRPC java generation process requires protoc 3 to create the java artifacts and gradle to drive the process.  
+
+Protoc can be installed via the instructions at: https://github.com/grpc/grpc-java#build-protobuf.  Installing it isn't always trivial; you may have to Google error messages to understand what to install next; there are plenty of answers on stackoverfolw for protobuf installation requirements.  
+
+Gradle version 2.3+ also has to be on your path; you can get it from https://gradle.org/downloads/.
+
+Once that's done, the easiest way to generate the java files is to run *gradle :generateProto*.  That will generate the java source files in target/generated-sources/main.  Manually delete src/main/generated-java/com and move  target/generated-sources/main/com to src/main/generated-java/.
```
Comment (kevinsi4508) [inline bigtable-protos/README.md]: remove the extra whitespace below:
"generated-java/com and move  target/gen"

**Comment by kevinsi4508** [comment]
LGTM

**Comment by AngusDavis** [comment]
My only comment is on the location of the generated code - grpc-java uses src/generated/java which I'm somewhat partial to (and seems to be supported by random StackOverflow commenters: http://stackoverflow.com/questions/22912900/location-of-generated-source-files-for-maven-directory-structure)

**Comment by sduskis** [comment]
I updated the directory structure along with the pom.xml file and README.

**Pull request merged**